config/default/traefik.yaml

```diff
@@ -0,0 +1,25 @@
+globalArguments:
+  - "--global.checknewversion"
+logs:
+  general:
+    format: json
+  access:
+    enabled: true
+    format: json
+providers:
+  kubernetesIngress:
+    publishedService:
+      enabled: true
+ingressRoute:
+  dashboard:
+    enabled: disabled
+tlsOptions:
+  default:
```

Comment:
```suggestion
  # https://ssl-config.mozilla.org/#server=traefik&version=2.4.5&config=intermediate&guideline=5.6
  default:
```

Thanks for sharing this awesome tool with us 🥳 
Do you mind adding it as a comment (as a contributor would search the code before the PRs comments)?

Reply:
They is always nice tools out there if you go look 😅 but I agree Mozilla did a great job. 👏